tests/containers/podman_netavark.pm

```diff
@@ -0,0 +1,143 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman, netavark, aardvark
+# Summary: Test podman netavark network backend
+# Maintainer: qac team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use version_utils qw(package_version_cmp is_transactional);
+use containers::utils qw(get_podman_version);
+use transactional qw(trup_call check_reboot_changes);
+use utils qw(zypper_call);
+use Utils::Systemd qw(systemctl);
+
+sub remove_subtest_setup {
+    assert_script_run("podman container rm -af");
+    assert_script_run("podman network prune -f");
+    assert_script_run("podman network ls");
+    assert_script_run("podman ps -a");
+}
+
+sub _cleanup {
+    my $podman = shift->containers_factory('podman');
+    select_console 'log-console';
+    remove_subtest_setup;
+    script_run('rm -rf /etc/containers/containers.conf');
+    $podman->cleanup_system_host();
+    validate_script_output('podman info --format {{.Host.NetworkBackend}}', sub { /cni/ });
+    validate_script_output('podman network ls', sub { /podman\s+bridge/ });
+}
+
+
+sub run {
+    my ($self, $args) = @_;
+
+    select_serial_terminal;
+    my $podman = $self->containers_factory('podman');
+    my $podman_version = get_podman_version();
+
+    if (package_version_cmp($podman_version, '4.4.2') < 0) {
+        record_info('No support', "Netavark backend is not supported in podman-$podman_version");
+        return 1;
+    }
+
+    my @pkgs = qw(netavark aardvark-dns);
+
+    if (is_transactional) {
+        trup_call("pkg install @pkgs");
+        check_reboot_changes;
+    } else {
+        zypper_call("in @pkgs");
+    }
+
+    $podman->cleanup_system_host();
+
+    ## change network backend to *netavark*
+    assert_script_run(q(echo -e '[Network]\nnetwork_backend="netavark"' > /etc/containers/containers.conf));
+    validate_script_output('podman info --format {{.Host.NetworkBackend}}', sub { /netavark/ });
+
+    ## TEST1
+    ## set static IP, and MAC addresses on a per-network basis
+    my $testnet = 'test_simple';
+    my $test_ctr = 'docker.io/fedora/apache';
+
+    assert_script_run("podman network create --gateway 10.90.0.1 --subnet 10.90.0.0/16 $testnet");
+    assert_script_run("podman run --network $testnet:ip=10.90.0.8,mac=76:22:33:44:55:66 -d --name test_apache $test_ctr");
+    assert_script_run("podman container inspect test_apache | grep $testnet");
```

Comment:
```suggestion
    assert_script_run("podman container inspect test_apache --format "{{.Network.Name}}");
```

Reply:
Have you tried this?

I don't see it there

```
 "NetworkSettings": {
               "EndpointID": "",
               "Gateway": "10.88.0.1",
               "IPAddress": "10.88.0.5",
               "IPPrefixLen": 16,
               "IPv6Gateway": "",
               "GlobalIPv6Address": "",
               "GlobalIPv6PrefixLen": 0,
               "MacAddress": "32:97:92:ae:2d:08",
               "Bridge": "",
               "SandboxID": "",
               "HairpinMode": false,
               "LinkLocalIPv6Address": "",
               "LinkLocalIPv6PrefixLen": 0,
               "Ports": {},
               "SandboxKey": "/run/netns/netns-535c4242-b557-5d18-abf1-ea582b666fef",
               "Networks": {
                    "podman": {
                         "EndpointID": "",
                         "Gateway": "10.88.0.1",
                         "IPAddress": "10.88.0.5",
                         "IPPrefixLen": 16,
                         "IPv6Gateway": "",
                         "GlobalIPv6Address": "",
                         "GlobalIPv6PrefixLen": 0,
                         "MacAddress": "32:97:92:ae:2d:08",
                         "NetworkID": "podman",
                         "DriverOpts": null,
                         "IPAMConfig": null,
                         "Links": null,
                         "Aliases": [
                              "e512738a8508"
                         ]
                    }
               }
          },
```

Reply:
```
assert_script_run("podman container inspect test_apache --format "{{ .NetworkSettings. Networks.podman. NetworkID }}");
```